ui/app/AppLayouts/CommunitiesPortal/CommunitiesPortalLayout.qml

```diff
@@ -34,164 +35,241 @@ StatusSectionLayout {
+                        if (localAccountSensitiveSettings.isDiscordImportToolEnabled) {
+                          Global.openPopup(chooseCommunityCreationTypePopupComponent)
+                        } else {
+                          Global.openPopup(createCommunitiesPopupComponent)
                         }
                     }
                 }
+            }
 
-                StatusBaseText {
-                    Layout.topMargin: 20
-                    text: qsTr("Popular")
-                    font.weight: Font.Bold
-                    font.pixelSize: d.subtitlePixelSize
-                    color: Theme.palette.directColor1
-                }
+            CommunityTagsRow {
+                tags: root.communitiesStore.communityTags
+                Layout.fillWidth: true
+            }
+
+            Item {
+                Layout.fillWidth: true
+                Layout.fillHeight: true
+                Layout.leftMargin: -d.preventShadowClipMargin
+                Layout.rightMargin: -d.preventShadowClipMargin
 
-                GridLayout {
-                    columns: 3
-                    columnSpacing: Style.current.padding
-                    rowSpacing: Style.current.padding
-
-                    Repeater {
-                        model: root.communitiesStore.curatedCommunitiesModel
-                        delegate: StatusCommunityCard {
-                            visible: !model.featured
-                            locale: communitiesStore.locale
-                            communityId: model.id
-                            loaded: model.available
-                            logo: model.icon
-                            name: model.name
-                            description: model.description
-                            members: model.members
-                            popularity: model.popularity
-                            // <out of scope> categories:  model.categories
-
-                            onClicked: { d.navigateToCommunity(communityId) }
+                clip: true
+
+                StatusScrollView {
+                    anchors.fill: parent
+                    anchors.rightMargin: d.preventShadowClipMargin
+                    anchors.leftMargin: d.preventShadowClipMargin
+
+                    padding: 0
+                    bottomPadding: d.layoutBottomMargin
+
+                    clip: false
+
+                    ColumnLayout {
+                        id: conentColumn
+
+                        StatusBaseText {
+                            id: featuredLabel
+                            visible: !d.searchMode
+                            Layout.topMargin: d.scrollViewTopMargin
+                            text: qsTr("Featured")
+                            font.weight: Font.Bold
+                            font.pixelSize: d.subtitlePixelSize
+                            color: Theme.palette.directColor1
+                        }
+
+                        GridLayout {
```

Comment:
this file is becoming overwhelming with more than 300LOC, could you try breaking it in smaller panels/views?

Reply:
good point, split into two files

Reply:
I smiled at 300 == overwhelming :)